sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -444,7 +444,7 @@ export class MessageSession extends LinkEntity<Receiver> {
       onMessageSettled(this.logPrefix, delivery, this._deliveryDispositionMap);
     };
 
-    this._notifyError = (args: ProcessErrorArgs) => {
+    this._notifyError = async (args: ProcessErrorArgs) => {
```

Comment:
Should the newly converted async methods be awaited when called?

Reply:
Updated